frontend/pages/admin/IntegrationsPage/cards/Vpp/VppSetupPage/components/DisableVppModal/DisableVppModal.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from "react";
+
+import Modal from "components/Modal";
+import Button from "components/buttons/Button";
+
+const baseClass = "diable-vpp-modal";
+
+interface IDisableVppModalProps {
+  onExit: () => void;
+}
+
+const DisableVppModal = ({ onExit }: IDisableVppModalProps) => {
+  const [isDisabling, setIsDisabling] = useState(false);
+
+  const onDisableVpp = () => {
+    // TODO: API integration
+  };
+
+  return (
+    <Modal
+      title="Disable Volume Purchasing Program (VPP)"
+      onExit={onExit}
+      className={baseClass}
+    >
+      <>
+        <p>
+          Apps purchased in Apple Business Manager won&apos;t appear in Fleet.
+          Apps won&apos;t be uninstalled from hosts. If you want to enable
+          integration again, you&apos;ll have to upload a new content token.
```

Comment:
> If you want to enable integration again

Is there a work missing here?

Reply:
i dont think so. what bit of work are you thinking?

Reply:
Seems like it should be "If you want to enable VPP integration again..." or "If you want to enable this integration again..." or something like that. Feels like "integration" is just hanging there and needs a modifier to link it to the prior sentence.

Reply:
ohhh sorry. ha, I though this said "a _work_ missing here?" yes it does sound a little strange.

Reply:
Lol, yeah, it did say "work". I edited it when I replied.

Reply:
@marko-lisica are you ok to change the copy to say "If you want to enable VPP integration again..." instead of "If you want to enable integration again"

Reply:
@ghernandez345 Sounds good to me. I'll update Figma as well.